universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam.csl

```diff
@@ -0,0 +1,427 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="symbol" et-al-min="1" demote-non-dropping-particle="sort-only" default-locale="id-ID">
+  <info>
+    <title>Universitas Negeri Semarang-Fakultas Matematika dan Ilmu Pengetahuan Alam</title>
```

Comment:
Could you change the title to `<title>Universitas Negeri Semarang - Fakultas Matematika dan Ilmu Pengetahuan Alam (Indonesian)</title>`?